oe_theme.theme

```diff
@@ -1208,6 +1208,11 @@ function oe_theme_preprocess_paragraph__oe_social_media_follow(array &$variables
     $variables['links'][$key]['label'] = $link['title'];
     $variables['links'][$key]['url'] = Url::fromUri($link['uri']);
   }
+  if (!empty($paragraph->get('field_oe_social_media_see_more')->getValue())) {
+    $other_link = $paragraph->get('field_oe_social_media_see_more')->getValue();
```

Comment:
$paragraph->get('field_oe_social_media_see_more')->first()->getValue() would be better in this case I believe.